Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -57,11 +57,7 @@ open class NavigationMapView: UIView {
     public var crossfadesCongestionSegments: Bool = false {
         didSet {
             if let routes = self.routes {
-                if routeLineTracksTraversal, let route = routes.first {
-                    setUpLineGradientStops(along: route)
-                } else {
-                    show(routes, legIndex: currentLegIndex)
-                }
+                show(routes, legIndex: currentLegIndex)
```

Comment:
Simplify the code when the route line properties change during active navigation by generating the new route line instead.